docs/notebooks/icnn_inits.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>the <code>NeuralDual</code> ... ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>?</p>

Reply:
<p>Introduced it differently.</p>

Reply:
<p>"in the context of the <code>NeuralDual</code> to compute" still sounds like an incomplete sentence.</p>

Reply:
:dizzy_face: